server/nodes/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+from .models import Node
+
+
+admin.site.register(Node)
```

Comment:
hmmm correct me if im wrong, does admin required to view the node?

Reply:
yeah, admin can add or remove nodes "to share with"